ci/long_running_tests/workloads/many_tasks_serialized_ids.py

```diff
@@ -1,5 +1,5 @@
 # This workload stresses distributed reference counting by passing and
-# returning serialized ObjectIDs.
+# returning serialized ObjectRefs.
```

Comment:
Please make sure to change all of the documentation (ray/doc/source/*.rst) too

Reply:
Sure. Will ping you again when I've changed everything.

Reply:
All python/cython/rst files should have been fixed